services/autoscaler-adapter/node_manager/utils.go

```diff
@@ -10,6 +12,163 @@ import (
+		"vcpu-8_memory-24g_nvidia-gtx-1080ti-2",
+		"vcpu-12_memory-36g_nvidia-gtx-1080ti-3",
+		"vcpu-16_memory-48g_nvidia-gtx-1080ti-4",
+		"vcpu-20_memory-60g_nvidia-gtx-1080ti-5",
+		"vcpu-24_memory-72g_nvidia-gtx-1080ti-6",
+		"vcpu-28_memory-84g_nvidia-gtx-1080ti-7",
+		"vcpu-32_memory-96g_nvidia-gtx-1080ti-8",
+
+		// Legacy CPU Instances AMD EPYC™
+		"vcpu-2_memory-4g_disk-80g",
+		"vcpu-4_memory-8g_disk-80g",
+		"vcpu-8_memory-16g_disk-80g",
+		"vcpu-12_memory-24g_disk-80g",
+		"vcpu-16_memory-32g_disk-80g",
+		"vcpu-20_memory-40g_disk-80g",
+		"vcpu-24_memory-48g_disk-80g",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ RTX 3090
+		"vcpu-4_memory-18g_disk-80g_nvidia3090-1",
+		"vcpu-8_memory-36g_disk-80g_nvidia3090-2",
+		"vcpu-12_memory-54g_disk-80g_nvidia3090-3",
+		"vcpu-16_memory-72g_disk-80g_nvidia3090-4",
+		"vcpu-20_memory-90g_disk-80g_nvidia3090-5",
+		"vcpu-24_memory-108g_disk-80g_nvidia3090-6",
+		"vcpu-28_memory-126g_disk-80g_nvidia3090-7",
+		"vcpu-32_memory-144g_disk-80g_nvidia3090-8",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ RTX 3090
+		"vcpu-4_memory-24g_disk-80g_nvidia3090-1",
+		"vcpu-8_memory-48g_disk-80g_nvidia3090-2",
+		"vcpu-12_memory-72g_disk-80g_nvidia3090-3",
+		"vcpu-16_memory-96g_disk-80g_nvidia3090-4",
+		"vcpu-20_memory-120g_disk-80g_nvidia3090-5",
+		"vcpu-24_memory-144g_disk-80g_nvidia3090-6",
+		"vcpu-28_memory-168g_disk-80g_nvidia3090-7",
+		"vcpu-32_memory-192g_disk-80g_nvidia3090-8",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ RTX 3090 - CPU & Memory Optimized
+		"vcpu-8_memory-48g_disk-80g_nvidia3090-1",
+		"vcpu-16_memory-96g_disk-80g_nvidia3090-2",
+		"vcpu-24_memory-144g_disk-80g_nvidia3090-3",
+		"vcpu-32_memory-192g_disk-80g_nvidia3090-4",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ RTX 3080
+		"vcpu-4_memory-12g_disk-80g_nvidia3080-1",
+		"vcpu-8_memory-24g_disk-80g_nvidia3080-2",
+		"vcpu-12_memory-36g_disk-80g_nvidia3080-3",
+		"vcpu-16_memory-48g_disk-80g_nvidia3080-4",
+		"vcpu-20_memory-60g_disk-80g_nvidia3080-5",
+		"vcpu-24_memory-72g_disk-80g_nvidia3080-6",
+		"vcpu-28_memory-84g_disk-80g_nvidia3080-7",
+		"vcpu-32_memory-96g_disk-80g_nvidia3080-8",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ RTX 3080 - CPU & Memory Optimized
+		"vcpu-8_memory-32g_disk-80g_nvidia3080-1",
+		"vcpu-16_memory-64g_disk-80g_nvidia3080-2",
+		"vcpu-24_memory-96g_disk-80g_nvidia3080-3",
+		"vcpu-32_memory-128g_disk-80g_nvidia3080-4",
+
+		// Legacy GPU Instances NVIDIA® GeForce™ GTX 1080 Ti
+		"vcpu-4_memory-12g_disk-80g_nvidia1080ti-1",
+		"vcpu-8_memory-24g_disk-80g_nvidia1080ti-2",
+		"vcpu-12_memory-36g_disk-80g_nvidia1080ti-3",
+		"vcpu-16_memory-48g_disk-80g_nvidia1080ti-4",
+		"vcpu-20_memory-60g_disk-80g_nvidia1080ti-5",
+		"vcpu-24_memory-72g_disk-80g_nvidia1080ti-6",
+		"vcpu-28_memory-84g_disk-80g_nvidia1080ti-7",
+		"vcpu-32_memory-96g_disk-80g_nvidia1080ti-8",
+	}
+
+	m := make(map[string]*typeInfo, len(instanceTypes))
+	for _, typ := range instanceTypes {
+		serverName := typ
```

Comment:
```suggestion
		serverName := type
```

has to be applied when the previous does

Reply:
type is a Go keyword.